### ISSUE TITLE
Mouse Constraint angular velocity seems backwards

### ISSUE THREAD

**Issue by mondospaceman** [liabru/matter-js#392]
I'd like to use a mouse constraint to rotate objects as well as pull them to a position.  I set angularStiffness to 0.2 and the objects do indeed rotate, but they seem to be rotating the wrong way.  I went through your demos and didn't find any that allowed a rotation in their mouse constraints.  I took the "getting started" example and added a constraint with angularStiffness = 0.2. 

If you click on the left side of a block and pull down, I expect the block to rotate counter clockwise, instead it rotates clockwise.

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Title</title>
</head>
<body>
<script src="./lib/matter.js"></script>
<script>
    // module aliases
    var Engine = Matter.Engine,
        Render = Matter.Render,
        World = Matter.World,
        Bodies = Matter.Bodies;

    // create an engine
    var engine = Engine.create();

    // create a renderer
    var render = Render.create({
        element: document.body,
        engine: engine
    });

    // create two boxes and a ground
    var boxA = Bodies.rectangle(400, 200, 80, 80);
    var boxB = Bodies.rectangle(450, 50, 80, 80);
    var ground = Bodies.rectangle(400, 610, 810, 60, { isStatic: true });

    // add all of the bodies to the world
    World.add(engine.world, [boxA, boxB, ground]);
```
I added the following:
```html
    engine.world.gravity.x = 0;
    engine.world.gravity.y = 0;
    var mouseConstraint = Matter.MouseConstraint.create(engine, {
        element: document.body,
        constraint: {
            stiffness: 0.2,
             angularStiffness: 0.2
        }
    });
    Matter.World.add(engine.world, mouseConstraint);
```
    // run the engine
    Engine.run(engine);

    // run the renderer
    Render.run(render);
</script>
</body>
</html>

**Comment by liabru** [comment]
See my comment in #254, does that seem to cover it? If so please +1 and I'll close this thanks.

**Comment by mondospaceman** [comment]
I'll check.  I think so, at least in accepting that it doesn't work!

**Comment by liabru** [comment]
Cool then I'll close this one then as I think it's the same problem, thanks.

**Comment by iwojap** [comment]
The connected issue doesn't fix this bug. This should be reopened.

**Comment by iwojap** [comment]
Fixed in 0.13.0
